hack/locate-manifests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/bash
+
+find ./ -name "manifest.yaml" -exec echo "---" \; -exec cat {} \; > extensions.yaml
```

Comment:
maybe just inline in the makefile itself?

Reply:
yeah, I thought the same now :) Already changed.